Straight_skeleton_2/test/Straight_skeleton_2/test_sls_offset.cpp

```diff
@@ -909,8 +909,10 @@ void test_offset(const char* filename)
 //    for(const auto& offp : offset_poly_with_holes)
 //      print_polygon_with_holes(*offp);
     CGAL::set_use_polygon_assertions(false);
-    for(const auto& offp : offset_poly_with_holes)
+    for(const auto& offp : offset_poly_with_holes){
+      (void)offp;
       assert(offp->outer_boundary().is_counterclockwise_oriented());
+    }
```

Comment:
I would write, using C++11 and C++14 features:
```suggestion
    using std::begin;
    using std::end;
    assert(std::all_of(begin(offset_poly_with_holes),
                       end(offset_poly_with_holes),
                       [](auto offp) {
                         return offp->outer_boundary().is_counterclockwise_oriented();
                       }));
```
`<algorithm>` and `iterator` must be included before.